app/src/sharedTest/java/org/oppia/android/app/parser/HtmlParserTest.kt

```diff
@@ -271,6 +271,71 @@ class HtmlParserTest {
     assertThat(htmlResult.toString()).endsWith(" ")
   }
 
+  @Test
+  fun testHtmlContent_changeDeviceToLtr_textViewDirectionIsSetToLtr() {
```

Comment:
Can you please confirm that the relevant tests fail here without the new fixes in place in HtmlParser?

Reply:
Added screenshots in PR description.